packages/client/wallets/smart-wallet/src/smartWalletService.ts

```diff
@@ -56,7 +58,17 @@ export interface UserParams {
     jwt: string;
 }
 
+export interface Callbacks {
+    onWalletCreationStarted?: () => Promise<void>;
```

Comment:
See previously defined callbacks [here](https://github.com/Crossmint/crossmint-sdk/commit/98d819729a917b88d333c5574b8726455681fe12#diff-c5100b9aa596950b4758a14a50a50f71c64f8b742801b6d3ea98e4f0a24a7febL107-L110)

I've generilzed them a bit (e.g. made them available for all signer types except passkeys only) to enable more use cases!